snc.sh

```diff
@@ -75,8 +75,8 @@ fi
 
 if [[ ${USE_PATCHED_RELEASE_IMAGE} == "enabled" ]]
 then
-   echo "Creating a new image with patched KAO/KCMO images"
-   create_new_release_with_patched_images
+   echo "Using release image with patched KAO/KCMO images"
+   OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=quay.io/crcont/ocp-release:${OPENSHIFT_RELEASE_VERSION}-${yq_ARCH}
```

Comment:
If we are using the `${OPENSHIFT_RELEASE_VERSION}-${yq_ARCH}` then what is the use of having single multiple arch image on the quay side? Did you test `openshift-install` make use of this multi arch image instead of arch specific one.

Reply:
I haven't tested `openshift-install` with an image manifest. Ideally we can delete the -arch images after the manifest is cretaed, but that depends on `openshift-install`. Even if it cannot use it, does not hurt to have the manifest in addition to the individual images.